cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -494,4 +494,13 @@
     /{,var/lib/snapd/}snap/core/*/usr/lib/snapd/snap-discard-ns rix,
     /var/lib/snapd/hostfs/{,var/lib/snapd/}snap/core/*/usr/lib/snapd/snap-discard-ns rix,
 
+    # As a special exception, allow HOME to be /var/lib/jenkins
+    /var/ r,
+    /var/lib/ r,
+    /var/lib/jenkins/ r,
+    /var/lib/jenkins/snap/{,*/,*/*/} rw,
```

Comment:
Same question here as in https://github.com/snapcore/snapd/pull/6446#discussion_r251833765

Reply:
Ah, SNAP_USER_DATA! Can you add a comment for that?

Reply:
Yes, in a followup